### PR TITLE
Cleanup values.yaml documentation

### PR DESCRIPTION
**Pull request by guidoiaquinti**
## Description
Goal is to make sure `helm-docs` runs cleanly and we have descriptions for most of the values. Mostly a cosmetic fix while I wait for CI tests to complete.

## Type of change
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)

## How has this been tested?
CI should be ✅ 
No values should have been added/removed (only comments).

## Checklist
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: charts/posthog/values.yaml [hunk 1]
```diff
@@ -1,100 +1,95 @@
...
   tag:
-  # -- Default image or tag, e.g. `:release-1.32.0` Do not overwrite, use image.sha or image.tag instead.
+  # -- PostHog default image. Do not overwrite, use `image.sha` or `image.tag` instead.
   default: ":release-1.32.0"
-  # -- Image pull policy
+  # -- PostHog image pull policy.
   pullPolicy: IfNotPresent
 
-# -- Required: Cloud service being deployed on. Either `gcp` or `aws` or `do` for DigitalOcean
-cloud:
```
Comment (guidoiaquinti) [inline charts/posthog/values.yaml:17]: Moved at the top

File: charts/posthog/values.yaml [hunk 2]
```diff
@@ -1,100 +1,95 @@
...
-    # -- CPU threshold percent for the events stack
+    # -- CPU threshold percent for the events stack HorizontalPodAutoscaler.
     cputhreshold: 60
-    # -- Min pods for the events stack
+    # -- Min pods for the events stack HorizontalPodAutoscaler.
     minpods: 1
-    # -- Max pods for the events stack
+    # -- Max pods for the events stack HorizontalPodAutoscaler.
     maxpods: 10
-  # -- Amount of events pods to run. Ignored if hpa is used
```
Comment (guidoiaquinti) [inline charts/posthog/values.yaml:46]: Moved above.

File: charts/posthog/values.yaml [hunk 3]
```diff
@@ -119,93 +115,91 @@ web:
...
+    # -- Min pods for the worker stack HorizontalPodAutoscaler.
     minpods: 1
-    maxpods: 20
+    # -- Max pods for the worker stack HorizontalPodAutoscaler.
+    maxpods: 10
+
+  # -- Additional env variables to inject into the worker stack deployment.
   env: []
-  # -- How many replicas of workers to run. Ignored if hpa is used
-  replicacount: 1
```
Comment (guidoiaquinti) [inline charts/posthog/values.yaml:133]: Moved above.

File: charts/posthog/values.yaml [hunk 4]
```diff
@@ -119,93 +115,91 @@ web:
...
-  # priorityClassName: ""
-  # schedulerName:
-  # Optional extra labels for pod, i.e. redis-client: "true"
-  # podLabels: []
-  # concurrency:
 
-# Outbound E-mails
 email:
-  # -- Outbound email sender
-  from_email:
```
Comment (guidoiaquinti) [inline charts/posthog/values.yaml:189]: moved below

**Review by hazzadous — APPROVED** [review]
Looks fine. I'm guessing I'd need to gen docs locally if I wanted to see the output?

**Comment by guidoiaquinti** [comment]
> Looks fine. I'm guessing I'd need to gen docs locally if I wanted to see the output?

Yes by running helm-docs (see the GH action for version and flags)

**Pull request merged**